modules/ROOT/pages/building-an-https-service.adoc

```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-CloudHub supports building HTTPS- and SSL-based services so you can be sure of your data confidentiality. 
-The configuration steps shown in this document are meant for deploying your Mule app to CloudHub.
+CloudHub supports deploying HTTPS- and SSL-based services so you ensure your data confidentiality. 
```

Comment:
Two things:

I was taught that open hyphenation causes translation difficulties, so consider "HTTPS-based and SSL-based services."

I was also taught that there is a legal issue with promising anything outright, including using "ensure" in such a context.

```suggestion
CloudHub supports deploying HTTPS-based and SSL-based services to help you ensure your data confidentiality. 
```